artview/components/plot_radar.py

```diff
@@ -849,6 +856,65 @@ def _update_fig_ax(self):
         self.ax.set_position([0.2, 0.55-0.5*yheight, xwidth, yheight])
         self.cax.set_position([0.2, 0.10, xwidth, 0.02])
         self._update_axes()
+    '''
+        options_type = [
+            ("Plot     top    left",  (float, float)),
+            ("Plot     button right", (float, float)),
```

Comment:
Could we change all of the "Plot" to "Plot area" ? This will also help with the spacing.